--- /config.def.h	2022-07-11 13:08:56.587105868 -0400
+++ /myconfig.def.h	2022-07-11 13:19:16.227069234 -0400
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Terminus:pixelsize=14:antialias=false:autohint=false";
 static int borderpx = 2;
 
 /*
